openslides/utils/autoupdate.py

```diff
@@ -1,165 +1,26 @@
 import json
-import os
-import posixpath
-from importlib import import_module
-from urllib.parse import unquote
 
-from django.conf import settings
-from django.core.wsgi import get_wsgi_application
-from sockjs.tornado import SockJSConnection, SockJSRouter
-from tornado.httpserver import HTTPServer
-from tornado.ioloop import IOLoop
-from tornado.options import parse_command_line
-from tornado.web import (
-    Application,
-    FallbackHandler,
-    HTTPError,
-    StaticFileHandler,
-)
-from tornado.wsgi import WSGIContainer
+from channels import Group
+from django.utils import timezone
 
-from ..users.auth import AnonymousUser, get_user
-from .access_permissions import BaseAccessPermissions
+from ..core.models import Session
+from ..users.auth import AnonymousUser
 
-RUNNING_HOST = None
-RUNNING_PORT = None
 
-
-class DjangoStaticFileHandler(StaticFileHandler):
-    """
-    Handels static data by using the django finders.
-
-    Only needed in the "small" version with tornado as wsgi server.
+def get_active_users():
     """
+    Helper to get all logged in users. Also returns the anonymous user if there
+    is at least one session with it.
```

Comment:
fixed by optimizing the query. Also renamed the function, because now it does never return the AnonymousUser